back-end/migrations/1574147599141_create_table_loan.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   up:
-    'CREATE TABLE IF NOT EXISTS loan (id INTEGER NOT NULL auto_increment primary key , userId INTEGER NOT NULL , productId INTEGER NOT NULL , remaining INTEGER NOT NULL);',
-  down: 'drop table loan'
+    'CREATE TABLE IF NOT EXISTS loans (id INTEGER NOT NULL auto_increment primary key , userId INTEGER NOT NULL , productId INTEGER NOT NULL , remaining INTEGER NOT NULL);',
```

Comment:
@amelieykw you shouldn't modify existing migrations, you just need to create a new one to modify the table (otherwise others would need to drop the database and create it again, which is fine since we don't have any data)